src/shared/wrapWdkDependencies.ts

```diff
@@ -16,13 +16,24 @@ export function areCompatibleWdkDependencies(
 }
 
 export function wrapWdkDependencies(
-  studyAccessApiUrl: string,
-  wdkDependencies: WdkDependencies
+  wdkDependencies: WdkDependencies,
+  studyAccessApiUrl?: string
```

Comment:
Nitpick:

```suggestion
  studyAccessApiUrl: string | undefined
  wdkDependencies: WdkDependencies,
```

Ordering the parameters this way makes it easier to use "partial application" to configure the WDK dependencies.

Reply:
So for legacy mbio, we'd configure `wrapWdkDependencies` by passing `partial(undefined)`.

Reply:
Thanks @jtlong3rd ! Ahh yes I see now the ordering works!

Out of curiosity, is the above a general best practices rule, so whenever i want to add `argument?: type` I should instead add `argument: type | undefined`? Or is it just best for this situation?

Reply:
In general, I would use the syntax `argument?: type` if I wanted `argument` to be optional, and the syntax `argument: type | undefined` if I wanted `argument` to be required (but to permit the value `undefined` to be passed).

The latter syntax was necessary for this situation (where you want `studyAccessApiUrl` to be the first parameter), because TypeScript doesn't allow an optional parameter to appear before a required parameter.